src/remotestorage.js

```diff
@@ -174,6 +174,9 @@ RemoteStorage.prototype = {
   /**
    * Initiate the OAuth authorization flow.
    *
+   * This function is usually not called by an app directly, but rather from a
+   * custom storage backend implementation (e.g. Dropbox or Google Drive).
+   *
```

Comment:
Is there actually an unusual case for using this directly? If not, shouldn't we make it private API?

Reply:
I don't know. Maybe if you want your app to be tied to one specific storage backend directly you might want use this function.

Reply:
I didn't mean just a general RS backend, but one specific server, e.g. a RS server that is specific to the app.

Reply:
I don't think I can follow. But I don't we should encourage people to use app-specific servers. It's also still not clear why one would use that function in this case.

Please let's be completely clear about this, and if there's no good case for using it then let's make it private and remove any confusing comments hinting at use cases without actually explaining a single one.

Reply:
Ok, I'll mark it as private and adapt the description.